kolibri/core/auth/utils/users.py

```diff
@@ -33,13 +34,9 @@ def get_remote_users_info(baseurl, facility_id, username, password):
              'users' containing the list of users of the facility if the user had rights.
     """
     user_info_url = reverse_remote(baseurl, "kolibri:core:publicuser-list")
-    params = {
-        "facility_id": facility_id,
-    }
     try:
         response = requests.get(
             user_info_url,
-            data=params,
```

Comment:
Was there a reason to remove this? @jredrejo had been reporting issues with this function being able to properly authenticate against a remote server as well, for the user merging flow, so wondering if there's something similar happening here?

Reply:
IIRC it was something around this `PublicFacilityUserViewset.get_queryset()` method that gets the `facility_id` from the query params since it's a `GET` -- in retrospect I think the actual issue is that `requests.get` doesn't take a `data` parameter it takes a `params` parameter so maybe this should just be `params=params`?

Reply:
Yep, that seems right! https://requests.readthedocs.io/en/latest/user/quickstart/#passing-parameters-in-urls

Reply:
seems like this still needs a resolution